src/GradingStudents/GradingStudents.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\GradingStudents;
+
+use InvalidArgumentException;
+
+class GradingStudents
+{
+    public function roundGrades($grades)
```

Comment:
> grades: an array of integers representing grades before rounding

O input deveria ser um array, não string. Aproveita, e adiciona um type hint aí ;)

Reply:
Acho que a parte da descrição da função acabou dando uma confundida nessa parte, no programa do hacker rank realmente é um array mas no *input format* mostra o formato certo que seria:
>The first line contains a single integer, **n**, the number of students.
Each line **i** of the **n** subsequent lines contains a single integer, **grades[i]**, denoting student 's grade.

Por isso o input está como string =s